src/sagemaker/estimator.py

```diff
@@ -3414,6 +3414,7 @@ def __init__(
         self.checkpoint_s3_uri = checkpoint_s3_uri
         self.checkpoint_local_path = checkpoint_local_path
         self.enable_sagemaker_metrics = enable_sagemaker_metrics
+        self.unsupported_dlc_image_for_sm_parallelism = ["2.0.1-gpu-py310-cu121"]
 
```

Comment:
add `2.0-gpu-py310-cu121` as we add other tags as well to the prod DLC

```
2.0-gpu-py310-cu121-ubuntu20.04-sagemaker-v2
```

Reply:
Added